docs/getting-started.md

```diff
@@ -20,12 +21,12 @@ Before you get started:
 
 > ⚠️ This project uses Go Modules. Due to the current state of code generation in [controller-tools](https://github.com/kubernetes-sigs/controller-tools) and [code-generator](https://github.com/kubernetes/code-generator), KUDO currently **must** be cloned into its `$GOPATH`-based location.
 
-## Installation Instructions
+## Install KUDO to your cluster
 
-- Get KUDO repo: `go get github.com/kudobuilder/kudo/`
+- `mkdir -p $GOPATH/src/github.com/kudobuilder/kudo`
 - `cd $GOPATH/src/github.com/kudobuilder/kudo`
-- `kubectl apply -f config/crds` to deploy the universal CRDs
-- `make run` to run the Operator with local go environment or `make deploy | kubectl apply -f -` to install to your kubernetes cluster
+- Get KUDO repo: `git clone git@github.com:kudobuilder/kudo.git`
+- `make deploy | kubectl apply -f -` to install controller and CRDs to your Kubernetes cluster
```

Comment:
add `cd kudo` step before `make deploy`

Reply:
why are we generating the CRDs to files under `config/crds` and committing to the repo if we expect users to `make deploy | kubectl apply -f -`?     I like the `make` approach.   should we delete the crds from the repo?